src/include/flow.h

```diff
@@ -467,6 +471,8 @@ class Flow {
         SendRst();
         state_ = State::kClosed;
         break;
+      case State::kRetryForRss:
+        [[fallthrough]];
```

Comment:
This will crash the stack with a `FATAL` error message, if someone tries to close the flow while it is in this state.

This is not the functionality we want to achieve.